tests/phpunit/unit-tests/admin/settings/class-llms-test-settings-engagements.php

```diff
@@ -10,6 +10,7 @@
  *
  * @since 3.37.3
  * @since 3.40.0 Add tests for `get_settings_group_email_delivery()`.
+ * @version [version]
```

Comment:
```suggestion
```

I know that I probably never updated our doc standards but we've talked about this before and we've stopped adding version tags in test class headers.

Reply:
Fine for me but I don't want to add a changelog entry there because it's already in the specific method doc.
Which is the reason why we stopped logging entries in the class doc blocks, do you agree?
So to me adding a version there was a good way to note the version of the tests.
Can I just remove that `version` and keep on not adding any entry in the file header?

Reply:
yes!